src/ILCompiler.Compiler/src/ILCompiler.Compiler.csproj

```diff
@@ -38,6 +38,7 @@
     <Compile Include="Compiler\CompilerTypeSystemContext.cs" />
     <Compile Include="Compiler\DelegateInfo.cs" />
     <Compile Include="Compiler\DependencyAnalysis\ArrayOfEmbeddedDataNode.cs" />
+    <Compile Include="Compiler\DependencyAnalysis\EETypeFlags.cs" />
```

Comment:
EETypeFlags should probably go under Common and we should then unify it with the enum that is currently duplicated under src/Runtime.Base/src/System/Runtime/EEType.cs.